editor/components/block-list/block.js

```diff
@@ -145,6 +145,10 @@ export class BlockListBlock extends Component {
 		) {
 			this.previousOffset = this.node.getBoundingClientRect().top;
 		}
+
+		if ( newProps.isTyping ) {
```

Comment:
Don't we need to compare with the previous prop or is unnecessary?

Reply:
We could, but it would have no impact, since `hideHoverEffects` guards against unnecessary `setState`, so even though this would repeatedly call `hideHoverEffects`, it would immediately abort because it's not hovered anyways.

Reply:
`hideHoverEffects` is itself a no-op if the state is already set

Reply:
Ah, GitHub didn't update fast enough to show Andrew's reply. :)